web/viewer.css

```diff
@@ -53,36 +53,9 @@ select {
   display: none !important;
 }
 
-#viewerContainer.pdfPresentationMode:-webkit-full-screen {
-  top: 0px;
-  border-top: 2px solid transparent;
-  background-color: #000;
-  width: 100%;
-  height: 100%;
-  overflow: hidden;
-  cursor: none;
-  -webkit-user-select: none;
-}
-
-#viewerContainer.pdfPresentationMode:-moz-full-screen {
-  top: 0px;
-  border-top: 2px solid transparent;
-  background-color: #000;
-  width: 100%;
-  height: 100%;
-  overflow: hidden;
-  cursor: none;
-  -moz-user-select: none;
-}
-
 #viewerContainer.pdfPresentationMode:-ms-fullscreen {
   top: 0px !important;
```

Comment:
This is the line I was mentioning in the description. The thing is, now that we have separate rules:

1. :-ms-fullscreen
2. :fullscreen

they get expanded by autoprefixer to:

1. :-ms-fullscreen
2. :-webkit-full-screen
3. :-moz-full-screen
4. :-ms-fullscreen
5. :fullscreen

As you can see, 1. was untouched, but 2 got expanded into all rules for all the browsers. So now, -ms-specific overwrites are before all the others, so if any other !important would occur in generic code, we can get these rules overwritten. This is not the case, thankfully, but it's something we should be aware of for the future.